src/utils/ScrollToTop.js

```diff
@@ -0,0 +1,9 @@
+import { useEffect } from 'react';
+
+export default function ScrollToTop() {
+  useEffect(() => {
+    window.scrollTo(0, 0);
+  }, []);
```

Comment:
I would suggest smooth scrolling :) 
```suggestion
  useEffect(() => {
    window.scrollTo(0, 0);
    behavior: 'smooth',
  }, []);
```